### PR TITLE
Query Helpers Updated

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Updates query helper and relevant test to use the latest Server Query format

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: templates/python/package/query.py.twig [hunk 1]
```diff
@@ -27,12 +27,36 @@ class Query:
     def search(attribute, value):
         return Query.addQuery(attribute, "search", value)
 
+    @staticmethod
+    def orderAsc(attribute):
+        return 'orderAsc("{}")'.format(attribute)
```
Comment (abnegate) [inline templates/python/package/query.py.twig:32]: We should use interpolation for all the python functions here

```suggestion
        return f'orderAsc("{attribute}")'
```

**Review by abnegate — COMMENTED** [review]
Looks good just some changes for python consistency :+1:

File: tests/Base.php [hunk 2]
```diff
@@ -59,11 +59,17 @@ abstract class Base extends TestCase
...
     const QUERY_HELPER_RESPONSES = [
-        'title.equal("Spiderman","Dr. Strange")',
-        'title.notEqual("Spiderman")',
-        'releasedYear.lesser(1990)',
-        'releasedYear.greater(1990,1999)',
-        'name.search("john")',
+        'equal("title", ["Spiderman","Dr. Strange"])',
+        'notEqual("title", ["Spiderman"])',
+        'lesser("releasedYear", [1990])',
+        'greater("releasedYear", [1990,1999])',
```
[the remark this answers is not in the document]
Comment (stnguyen90) [reply tests/Base.php:65]: FYI, `greater("releasedYear", [1990,1999])` doesn't really make sense.

File: templates/kotlin/src/main/kotlin/io/appwrite/Query.kt.twig [hunk 3]
```diff
@@ -6,20 +6,32 @@ class Query {
...
     fun search(attribute: String, value: String) = Query.addQuery(attribute, "search", value)
 
-    private fun addQuery(attribute: String, oper: String, value: Any): String {
+    fun orderAsc(attribute: String) = "orderAsc(\"${attribute}\")"
+
+    fun orderDesc(attribute: String) = "orderDesc(\"${attribute}\")"
+
+    fun cursorBefore(attribute: String) = "cursorBefore(\"${attribute}\")"
+
+    fun cursorAfter(attribute: String) = "cursorAfter(\"${attribute}\")"
```
Comment (stnguyen90) [inline templates/kotlin/src/main/kotlin/io/appwrite/Query.kt.twig:25]: parameters should be documentId

File: templates/node/index.d.ts.twig [hunk 4]
```diff
@@ -147,13 +147,13 @@ declare module "{{ language.params.npmPackage|caseDash }}" {
...
+    static lessThan(attribute: string, value: QueryTypes): string;
 
-    static lesserEqual(attribute: string, value: QueryTypes): string;
+    static lessThanEqual(attribute: string, value: QueryTypes): string;
 
-    static greater(attribute: string, value: QueryTypes): string;
+    static greaterThan(attribute: string, value: QueryTypes): string;
 
-    static greaterEqual(attribute: string, value: QueryTypes): string;
+    static greaterThanEqual(attribute: string, value: QueryTypes): string;
```
Comment (stnguyen90) [inline templates/node/index.d.ts.twig:156]: This file is missing new methods.

File: templates/php/src/Query.php.twig [hunk 5]
```diff
@@ -88,17 +88,77 @@ class Query
...
+        return 'cursorAfter("' . $attribute . '")';
+    }
+
+    /**
+     * Cursor Before
+     *
+     * @param String $attribute
+     * @return string
+     */
+    public static function cursorBefore(String $attribute): string {
```
Comment (stnguyen90) [inline templates/php/src/Query.php.twig:107]: parameter should be documentId

File: templates/php/src/Query.php.twig [hunk 6]
```diff
@@ -88,17 +88,77 @@ class Query
...
+        return 'orderAsc("' . $id . '")';
+    }
+
+    /**
+     * Order Desc
+     *
+     * @param String $id
+     * @return string
+     */
+    public static function orderDesc(String $id): string {
```
Comment (stnguyen90) [inline templates/php/src/Query.php.twig:127]: parameters should be attribute

File: tests/Base.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (christyjacob4) [reply tests/Base.php:65]: yes, updated them as well 👍

File: templates/android/library/src/main/java/io/appwrite/Query.kt.twig [hunk 7]
```diff
@@ -6,20 +6,32 @@ class Query {
...
     fun search(attribute: String, value: String) = Query.addQuery(attribute, "search", value)
 
-    private fun addQuery(attribute: String, oper: String, value: Any): String {
+    fun orderAsc(attribute: String) = "orderAsc(\"${attribute}\")"
+
+    fun orderDesc(attribute: String) = "orderDesc(\"${attribute}\")"
+
+    fun cursorBefore(attribute: String) = "cursorBefore(\"${attribute}\")"
+
+    fun cursorAfter(attribute: String) = "cursorAfter(\"${attribute}\")"
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply templates/android/library/src/main/java/io/appwrite/Query.kt.twig:25]: updated

File: templates/kotlin/src/main/kotlin/io/appwrite/Query.kt.twig (hunk 3 above)
Comment (christyjacob4) [reply templates/kotlin/src/main/kotlin/io/appwrite/Query.kt.twig:25]: updated

File: templates/node/index.d.ts.twig (hunk 4 above)
Comment (christyjacob4) [reply templates/node/index.d.ts.twig:156]: added them

File: templates/php/src/Query.php.twig (hunk 5 above)
Comment (christyjacob4) [reply templates/php/src/Query.php.twig:107]: done 👍

File: templates/php/src/Query.php.twig (hunk 6 above)
Comment (christyjacob4) [reply templates/php/src/Query.php.twig]: updated

**Pull request merged**